lang/fpc/Portfile

```diff
@@ -220,12 +220,15 @@ subport "${name}-cross-x86-64-win64" {
     }
 }
 
-if {${subport} eq "${name}"} {
+if {${subport} == "${name}"} {
```

Comment:
I think `eq` is in fact correct.

Reply:
I had a mix of eq and == and wanted to have one only.

Reply:
`==` is used for numerical values, `eq` for strings.

Reply:
oh. I see.

Reply:
So can we fix this please?
```suggestion
if {${subport} eq "${name}"} {
```

Reply:
done